### ISSUE TITLE
Missing component info for "RSGPalette"

### ISSUE THREAD

**Issue by TwitchBronBron** [rokucommunity/brighterscript#1057]
![image](https://github.com/rokucommunity/brighterscript/assets/2544493/f59f926c-8e5a-4684-8e9a-152ae214d204)

**Comment by markwpearce** [comment]
There is no roku docs page for this... we need to add it manually

**Comment by TwitchBronBron** [comment]
Here are all the docs pages that seem to reference it: 
- https://developer.roku.com/en-ca/docs/references/scenegraph/dynamic-voice-keyboard-nodes/dynamic-key-grid.md
- https://developer.roku.com/en-ca/docs/references/scenegraph/scene.md
- https://developer.roku.com/en-ca/docs/references/scenegraph/standard-dialog-framework-nodes/std-dlg-action-card-item.md
- https://developer.roku.com/en-ca/docs/references/scenegraph/standard-dialog-framework-nodes/standard-dialog.md
- https://developer.roku.com/en-ca/docs/references/scenegraph/standard-dialog-framework-nodes/standard-dialog-framework-overview.md

The docs are a little confusing, I think the palette might actually just have a single AA property called `colors`. 

```brighterscript
    m.top.palette = createObject("roSGNode", "RSGPalette")

    m.top.palette.colors = {    DialogBackgroundColor: "0x002040FF",
                                DialogItemColor: "0x007FEFFF",
'                                DialogTextColor: "0x007FEFFF",
                                DialogTextColor: "0xC0C0C0FF",
'                                DialogFocusColor: "0x007FEFFF",
                                DialogFocusColor: "0xC0C0C0FF",
                                DialogFocusItemColor: "0x003E7EFF",
                                DialogSecondaryTextColor: "0x007FEF66",
                                DialogSecondaryItemColor: "0x807FFF4D",
                                DialogInputFieldColor: "0x807FFF80",
                                DialogKeyboardColor: "0x807FFF4D",
                                DialogFootprintColor: "0x807FFF4D" }

    dialogTextColor = m.top.palette.colors["DialogTextColor"]
```

And here is a summary of all the the AA properties across the docs:
- **KeyboardColor:**	Blend color for key background bitmap.
- **PrimaryTextColor:**	Text color used for non-focused keys. Blend color for the icons of non-focused keys. Text color for the label of focused key suggestion items.
- **SecondaryItemColor:**	Text color for disabled keys. Blend color for the icons of disabled keys.
- **FocusColor:**	Blend color for the focus indicator. Blend color for the background of key suggestion pop-us.
- **FocusItemColor:**	Text color for the label of the focused key. Blend color for the icons of the focused key and the focus indicator in key suggestion pop-ups. Text color for the labels of non-focused key suggestion items.


- **PrimaryTextColor:**	The text color used for the entered text displayed in the VoiceTextEditBox node.
- **SecondaryTextColor:**	The text color used for the hints displayed in the VoiceTextEditBox.
- **InputFieldColor:**	The blend color applied to the VoiceTextEditBox background
- **DialogBackgroundColor:**	Blend color for dialog's background bitmap.
- **DialogItemColor:**	Blend color for the following items:
    - StdDlgProgressItem's spinner bitmap
    - StdDlgDeterminateProgressItem's graphic
- **DialogTextColor:**	Color for the text in the following items:
    - StdDlgTextItem and StdDlgGraphicItem if the namedTextStyle field is set to "normal" or "bold".
    - All content area items, except for StdDlgTextItem and StdDlgGraphicItem.
    - Title area. Unfocused button.
- **DialogFocusColor:**	Blend color for the following:
    - The button area focus bitmap.
    - The focused scrollbar thumb.
- **DialogFocusItemColor:**	Color for the text of the focused button.
- **DialogSecondaryTextColor:**	Color for the text of in the following items:
    - StdDlgTextItem and StdDlgGraphicItem if the namedTextStyle field is set to "secondary".
    - Disabled button.
- **DialogSecondaryItemColor:**	Color for the following items:
    - The divider displayed below the title area.
    - The unfilled portion of the StdDlgDeterminateProgressItem's graphic.
- **DialogInputFieldColor:**	The blend color for the text edit box background bitmap for keyboards used inside dialogs.
- **DialogKeyboardColor:**	The blend color for the keyboard background bitmap for keyboards used inside dialogs
- **DialogFootprintColor:**	The blend color for the following items:
    - The button focus footprint bitmap that is displayed when the button area does not have focus.
    - Unfocused scrollbar thumb and scrollbar track.

**Comment by markwpearce** [comment]
Fixed with #1065
